### ISSUE TITLE
Number of epochs

### ISSUE THREAD

**Issue by asifahmad7007** [DIAGNijmegen/neural-odes-segmentation#2]
Hi, I have reduced the number of epochs from 100 (as you have chosen) to 05 , as it takes much more time on GOOGLE Colab. It does not stop after completion of 5th epoch and has exceeded to further epochs. Still running! What could be the issue? 
I have made changes in:
`accumulate_batch = 8  # mini-batch size by gradient accumulation
accumulated = 0

if TRAIN_RESNET: filename = 'best_border_resnet_model.pt'
elif TRAIN_UNODE: filename = 'best_border_unode_model.pt'
elif TRAIN_UNET: filename = 'best_border_unet_model.pt'

def run(lr=1e-3, epochs=5):
    accumulated = 0
    for param_group in optimizer.param_groups:
        param_group['lr'] = lr`

And what does this mean: 
`run(lr, 600 - len(losses))`

Above you had selected `epochs = 100` I changed it to `epochs=5`

Confused! 
Would be very thankful if you guide in this regard.

**Comment by HansPinckaers** [comment]
Hi,

```run``` is a (badly-named) function that runs the experiment for a configured number of epochs. You changed the default to be five epochs. But the cell function is called with ```epochs=600```-argument, which overrides the default.

If you want less epochs change
```run(lr, 600 - len(losses))``` to ```run(lr, 5 - len(losses))```

``` len(losses)``` is substracted in case you are resuming and already trained some epochs.

**Comment by asifahmad7007** [comment]
Thank you so much! That confused me: `epochs = 100` and then `600`. 600 overrides the 100!
Got it! thanks a lot!
